src/equations/compressible_euler_multicomponent_2d.jl

```diff
@@ -0,0 +1,714 @@
+
+@doc raw"""
+    CompressibleEulerMulticomponentEquations2D
+
+  !!! warning "Experimental code"
```

Comment:
```suggestion
!!! warning "Experimental code"
```
Otherwise I fear it might not get rendered properly by the markdodwn parser.

Reply:
fixed in 9424975